specification/hugr.md

```diff
@@ -769,17 +772,18 @@ indices after the list of node indices?
 ## Type System
 
 There are three classes of type: ``AnyType`` $\supset$ ``CopyableType`` $\supset$ ``EqType``. Types in these classes are distinguished by the operations possible on (runtime) values of those types:
-  - For the broadest class (``AnyType``), the only operation supported is the identity operation (aka no-op, or `lift` - see [Extension Tracking](#extension-tracking) below). Specifically, we do not require it to be possible to copy or discard all values, hence the requirement that outports of linear type must have exactly one edge. (That is, a type not known to be in the copyable subset). All incoming ports must have exactly one edge.
 
-    In fully qubit-counted contexts programs take in a number of qubits as input and return the same number, with no discarding. See [quantum extension](#quantum-extension) for more.
+- For the broadest class (``AnyType``), the only operation supported is the identity operation (aka no-op, or `lift` - see [Extension Tracking](#extension-tracking) below). Specifically, we do not require it to be possible to copy or discard all values, hence the requirement that outports of linear type must have exactly one edge. (That is, a type not known to be in the copyable subset). All incoming ports must have exactly one edge.
+
+    In fully qubit-counted contexts programs take in a number of qubits as input and return the same number, with no discarding.
 
-  - The next class is ``CopyableType``, i.e. types holding ordinary classical
-    data, where values can be copied (and discarded, the 0-ary copy). This
-    allows multiple (or 0) outgoing edges from an outport; also these types can
-    be sent down Static edges. Note: dataflow inputs (Value and Static) always
-    require a single connection.
+- The next class is ``CopyableType``, i.e. types holding ordinary classical
+  data, where values can be copied (and discarded, the 0-ary copy). This
+  allows multiple (or 0) outgoing edges from an outport; also these types can
+  be sent down `Static` edges. Note: dataflow inputs (`Value` and `Static`) always
```

Comment:
I wonder if we can standardize on single backticks throughout instead of double backticks? I think the only reason to use double backticks is if the text contains single-backtick characters (which are then rendered verbatim), but in most (all?) case here that doesn't apply. But maybe this would be best done in a separate PR.

Reply:
Fixed.